equinix/fabric_mapping_helper.go

```diff
@@ -252,6 +264,27 @@ func locationToFabric(locationList []interface{}) v4.SimplifiedLocation {
 	return sl
 }
 
+func virtualdeviceToFabric(virtualdeviceList []interface{}) v4.VirtualDevice {
+	vd := v4.VirtualDevice{}
+	for _, ll := range virtualdeviceList {
+		llMap := ll.(map[string]interface{})
+		tp := llMap["type"].(string)
```

Comment:
`name` was added to the VD schema but it isn't being parsed into the model here.